api-metastore/src/main/java/org/zalando/nakadi/service/SubscriptionsUriHelper.java

```diff
@@ -10,13 +12,17 @@
 public class SubscriptionsUriHelper {
 
     public static PaginationLinks.Link createSubscriptionListLink(
-            final Optional<String> owningApplication, final Set<String> eventTypes, final int offset,
+            final Optional<String> owningApplication, final Set<String> eventTypes,
+            final Optional<AuthorizationAttribute> reader, final int offset,
             final Optional<SubscriptionTokenLister.Token> token, final int limit, final boolean showStatus) {
 
         final UriComponentsBuilder urlBuilder = UriComponentsBuilder.fromPath("/subscriptions");
         if (!eventTypes.isEmpty()) {
             urlBuilder.queryParam("event_type", eventTypes.toArray());
         }
+        reader.ifPresent(_reader -> {
```

Comment:
`_reader ` is `_` conventional notation for lambda vars?

Reply:
Nope, I just didn't have a better choice.

Reply:
in the project we use lambdas without underscore, i suggest we keep it consistent